data/styles/clear/include/Roads.mapcss

```diff
@@ -814,6 +836,23 @@ line|z18[highway=steps]
 line|z19-[highway=steps]
 {width: 10;dashes: 4,4;}
 
+line|z16[highway=steps][bridge?]::bridgewhite
+{casing-width-add: -2;}
+line|z17[highway=steps][bridge?]::bridgewhite
+{casing-width-add: -2.5;}
+line|z18[highway=steps][bridge?]::bridgewhite
+{casing-width-add: -3;}
+line|z19-[highway=steps][bridge?]::bridgewhite
+{casing-width-add: -3.5;}
+line|z16[highway=steps][bridge?]::bridgeblack
+{casing-width-add: -1.5;}
+line|z17[highway=steps][bridge?]::bridgeblack
+{casing-width-add: -2;}
+line|z18[highway=steps][bridge?]::bridgeblack
+{casing-width-add: -2.5;}
+line|z19-[highway=steps][bridge?]::bridgeblack
+{casing-width-add: -3;}
```

Comment:
do stairs actually fit into the bridge outline with these params? :)
would be great to see a screenshot!

Reply:
There are screenshots of https://www.openstreetmap.org/way/759824860 in the first post